README.md

```diff
@@ -107,6 +106,35 @@ To run everything, run:
 make check
 ```
 
+## Comparison of developer environments
+
+### Developer environment on non-Qubes
+
+* Ran by `run.sh`
+* Uses temporary configuration directories by default
+* Requests/responses to the Journalist API are sent directly via HTTP (and Tor is not used)
+* Does not use `split-gpg`
+
+### Developer environment on Qubes
+
+* Ran by directly invoking the client `python -m securedrop_client`.
+* Requires that `make all` in the `securedrop-workstation` repository has completed successfully
+* Uses `~/.securedrop_client` as its configuration directory
```

Comment:
* you could mention to run securedrop_client in the `sd-svs` vm and that the developer needs to set networking to sys-firewall
* might also be helpful to somewhere link to the securedrop-workstation docs on running the client in qubes: https://github.com/freedomofpress/securedrop-workstation/#using-the-securedrop-client. I'm just now noticing that in these docs we point to https://docs.securedrop.org/en/latest/development/setup_development.html?highlight=development#qubes for setting up the dev environment. It might be easier for new devs if we just have complete instructions here rather than the three different locatoins.